lab_2_keywords_cooccurrence/main.py

```diff
@@ -29,7 +37,33 @@ def extract_candidate_keyword_phrases(phrases: Sequence[str], stop_words: Sequen
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not isinstance(phrases, list):
+        return None
+    if not isinstance(stop_words, list):
+        return None
+    if stop_words == [] or phrases == []:
```

Comment:
this is a bad way to check whether the object is empty
consider doing it as `if not tokens`